lint-server error message

When running `make install` from master we are getting this lint error:
```
elyra/pipeline/tests/util.py:35:1: H238: old style class declaration, use new style (inherit from `object`)
elyra/pipeline/tests/resources/node_util/node_util.py:23:1: H238: old style class declaration, use new style (inherit from `object`)
make: *** [lint-server] Error 1
```

--- a/elyra/pipeline/tests/resources/node_util/node_util.py
+++ b/elyra/pipeline/tests/resources/node_util/node_util.py
@@ -20,7 +20,7 @@ from abc import ABC, abstractmethod
 from typing import List
 
 
-class NodeFile():
+class NodeFile(object):
     """Base class for input and output node files"""
     def __init__(self, filename: str) -> None:
         self.filename = filename

--- a/elyra/pipeline/tests/util.py
+++ b/elyra/pipeline/tests/util.py
@@ -32,7 +32,7 @@ def _read_pipeline_resource(pipeline_filename):
     return pipeline_json
 
 
-class NodeBase:
+class NodeBase(object):
     """Represents a node of a constructed pipeline based on files in resources/node_util. """
     id: str
     name: str

